src/mappers.ts

```diff
@@ -166,17 +167,31 @@ function validateName(value:string)
     var r = new RegExp(/[^a-zA-Z0-9_-]/);
     return r.test(value);
 }
+function validatedisplayName(value : string){
+    if(value.length < 2 || value.length > 50) return true;
+    return false;
+ }
 export async function getInputParams() {
     await getAccessToken("https://management.core.windows.net");
     YamlPath = core.getInput('loadTestConfigFile');
     if(!(YamlPath.includes(".yaml") || YamlPath.includes(".yml")))
         throw new Error("The Load Test configuration file should be of type .yaml or .yml");
     const config = yaml.load(fs.readFileSync(YamlPath, 'utf8'));
-    if(config.testName == null || config.testName == undefined)
-        throw new Error("The required field testName is missing in "+YamlPath+".");
-    testName = (config.testName).toLowerCase();
-    if(validateName(testName))
-        throw new Error("Invalid testName. Allowed chararcters are [a-zA-Z0-9-_]");
+    if((config.testName == null || config.testName == undefined) && (config.testId == null || config.testId == undefined))
+        throw new Error("The required field testId is missing in "+YamlPath+".");
+    if(config.testName != null && config.testName != undefined){
+        testId = (config.testName).toLowerCase();
+    }
+    if(config.testId != null && config.testId != undefined){
+        testId = (config.testId).toLowerCase();
+    }
+    displayName = testId;
+    if(config.displayName != null && config.displayName != undefined)
+        displayName = config.displayName;
+    if(validateName(testId))
+        throw new Error("Invalid testId. Allowed chararcters are [a-z0-9-_]");
+    if(validatedisplayName(displayName))
+        throw new Error("Invalid display name.Display name cannot contain special characters @!`~+^, or begin or end with '_' or '-'.");
```

Comment:
Can we change this to display name must be between 2 to 50 characters since that is only being validated